packages/caip/src/adapters/banxa/index.ts

```diff
@@ -57,9 +69,10 @@ export const getSupportedBanxaAssets = () =>
  * chain won't be exactly the same as ours.
  */
 const chainIdToBanxaBlockchainCodeMap: Record<ChainId, string> = {
-  'eip155:1': 'ETH',
-  'bip122:000000000019d6689c085ae165831e93': 'BTC',
-  'cosmos:cosmoshub-4': 'COSMOS'
+  [ethChainId]: 'ETH',
+  [btcChainId]: 'BTC',
+  [cosmosChainId]: 'COSMOS',
+  [avalancheChainId]: 'AVAX'
```

Comment:
`AVAX` is the default and now points to c-chain (previously was pointing at x-chain). They also have an `AVAX-C` blockchain that looks to route correctly to c-chain now that it has been enabled. Wondering if we want to use `AVAX-C` explicitly in case their default blockchain changes?

Main thing that caught my attention is `AVAX` used to route to x-chain, but doesn't anymore so maybe it doesn't matter at this point.
```suggestion
  [avalancheChainId]: 'AVAX-C'
```

Reply:
> Main thing that caught my attention is `AVAX` used to route to x-chain, but doesn't anymore so maybe it doesn't matter at this point.

That would be why, C-Chain support has been enabled for us: https://discord.com/channels/554694662431178782/885616186623148062/1007460513329983608

Reply:
https://docs.google.com/spreadsheets/d/1KU6J1Hl4vxBTIWCwWdrFfSadNltuFheQRoJyPrd0LMQ/edit#gid=631982242

Reply:
Good point about the C-Chain, we should probably err on the side of safety and use this one, more declarative too

Reply:
> > Main thing that caught my attention is `AVAX` used to route to x-chain, but doesn't anymore so maybe it doesn't matter at this point.
> 
> That would be why, C-Chain support has been enabled for us: https://discord.com/channels/554694662431178782/885616186623148062/1007460513329983608

Yeah I saw that which prompted me to test. I just don't know if they would re-enable x-chain and that would become the default for `blockchain=AVAX`, or if it is just how we tell them to set it up for our account. `AVAX-C` would just be the most explicit in case anything on backend changed possibly is I guess what I'm getting at